DfsPlugin/src/org/workcraft/plugins/dfs/DfsToolsProvider.java

```diff
@@ -18,7 +18,7 @@
     public Iterable<GraphEditorTool> getTools() {
         ArrayList<GraphEditorTool> result = new ArrayList<>();
 
-        result.add(new SelectionTool(false));
```

Comment:
The bool is not used in the receiver, so not really. I removed it from there too. If that wasn't the case, this wouldn't compile.